### PR TITLE
update readme w/wiki links and more info

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,406 +1,75 @@
...
-- [ASP.NET Core Web app incremental tutorial](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2) chapter 2.2, [call Microsoft Graph on behalf of a signed in user with a SQL token cache](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/tree/master/2-WebApp-graph-user/2-2-TokenCache)
-- [Web app that calls web apis](https://docs.microsoft.com/azure/active-directory/develop/scenario-web-app-call-api-overview) scenario overview in the Microsoft identity platform documentation, and related articles
-
-## Web APIs
+[Microsoft Identity Web](https://www.nuget.org/packages/Microsoft.Identity.Web) is a library which contains a set of reusable classes used in conjunction with ASP .NET Core for integrating with the [Microsoft identity platform](https://docs.microsoft.com/en-us/azure/active-directory/develop/) (formerly *Azure AD v2.0 endpoint*) and [AAD B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/).
```
Comment (pmaytak) [inline README.md:3]: `ASP .NET` > `ASP.NET`

File: README.md [hunk 2]
```diff
@@ -1,406 +1,75 @@
...
+- Microsoft Identity Web is expected to become Generally Available in Fall 2020.
 
-To troubleshoot your web API, you can set the `subscribeToJwtBearerMiddlewareDiagnosticsEvents` optional boolean to `true` when you call `AddProtectedWebApi`. Enabling these diagnostics displays in the output window the progression of the OAuth 2.0 message through the JWTBearer middleware (from the reception of the message from Azure Active directory to the availability of the user identity in `HttpContext.User`).
+## Using Microsoft Identity Web
 
-<img alt="JwtBearerMiddlewareDiagnostics" src="https://user-images.githubusercontent.com/13203188/62538382-7d6ba800-b807-11e9-9540-560e7129197b.png" width="65%"/>
+- The conceptual documentation is currently available from the [Wiki pages](https://github.com/AzureAD/microsoft-identity-web/wiki).
+- Code samples are available:
+  > [Web app samples](https://github.com/AzureAD/microsoft-identity-web/wiki/web-app-samples)
```
Comment (pmaytak) [inline README.md:46]: `app` > `App`?

File: README.md [hunk 3]
```diff
@@ -1,406 +1,75 @@
...
+  > [Web API samples](https://github.com/AzureAD/microsoft-identity-web/wiki#web-api-samples)
```
Comment (pmaytak) [inline README.md:47]: Put these code links on the separate lines or just put in one line:

Code samples are available for [Web Apps](https://github.com/AzureAD/microsoft-identity-web/wiki/web-app-samples) and [Web APIs](https://github.com/AzureAD/microsoft-identity-web/wiki#web-api-samples).

File: README.md [hunk 4]
```diff
@@ -1,406 +1,75 @@
...
 
-In both cases, you can set a breakpoint in the methods of the  `OpenIdConnectMiddlewareDiagnostics` and `JwtBearerMiddlewareDiagnostics` classes respectively to observe values in the debugger.
+## Where do I file issues
 
-## More customization
+This is the correct repo to file [issues](https://github.com/AzureAD/microsoft-authentication-identity-web/issues)
```
Comment (pmaytak) [inline README.md:51]: Missing period at the end.

File: README.md [hunk 5]
```diff
@@ -1,406 +1,75 @@
...
-token validation by reading:
+If you find a security issue with our libraries or services please report it to [secure@microsoft.com](mailto:secure@microsoft.com) with as much detail as possible. Your submission may be eligible for a bounty through the [Microsoft Bounty](http://aka.ms/bugbounty) program. Please do not post security issues to GitHub Issues or any other public site. We will contact you shortly upon receiving the information. We encourage you to get notifications of when security incidents occur by visiting [this page](https://technet.microsoft.com/en-us/security/dd252948) and subscribing to Security Advisory Alerts.
 
-- [Validating Tokens](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/ValidatingTokens) in that library's conceptual documentation
-- [TokenValidationParameters](https://docs.microsoft.com/dotnet/api/microsoft.identitymodel.tokens.tokenvalidationparameters?view=azure-dotnet)'s reference documentation
+Copyright (c) Microsoft Corporation.  All rights reserved. Licensed under the MIT License (the "License")
```
Comment (pmaytak) [inline README.md:75]: Missing period at the end.

File: README.md [hunk 6]
```diff
@@ -1,406 +1,75 @@
...
-- [Token cache serialization](msal-net-token-cache-serialization)
+## Security Reporting
 
-Token validation is performed by the classes in the [Identity Model Extensions for .NET](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet) library. Learn about customizing
-token validation by reading:
+If you find a security issue with our libraries or services please report it to [secure@microsoft.com](mailto:secure@microsoft.com) with as much detail as possible. Your submission may be eligible for a bounty through the [Microsoft Bounty](http://aka.ms/bugbounty) program. Please do not post security issues to GitHub Issues or any other public site. We will contact you shortly upon receiving the information. We encourage you to get notifications of when security incidents occur by visiting [this page](https://technet.microsoft.com/en-us/security/dd252948) and subscribing to Security Advisory Alerts.
```
Comment (pmaytak) [inline README.md:73]: `or services please report` > `or services, please report`. (Microsoft Word says there should be a comma.)

File: README.md [hunk 7]
```diff
@@ -1,406 +1,75 @@
...
 
-```
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
-## Learn more about the library
+## Security Library
 
-You can learn more about the tokens by looking at the following articles in MSAL.NET's conceptual documentation:
+This library controls how users sign-in and access services. We recommend you always take the latest version of our library in your app when possible. We use [semantic versioning](http://semver.org) so you can control the risk associated with updating your app. As an example, always downloading the latest minor version number (e.g. x.*y*.x) ensures you get the latest security and feature enhancements but our API surface remains the same. You can always see the latest version and release notes under the Releases tab of GitHub.
```
Comment (pmaytak) [inline README.md:69]: `feature enhancements but our API` > `feature enhancements, but our API`. (Microsoft Word says there should be a comma.)

**Comment by pmaytak** [comment]
@jennyf19 to please look over my feedback.

File: README.md (hunk 2 above)
Comment (jmprieur) [reply README.md:46]: no, actually the doc people recomment "Web app", and "Web API"

**Comment by jmprieur** [comment]
PR comments address, @jennyf19 (in master directly as these were typos)

**Pull request merged**